dnd5esheets/front/src/components/CharacterSheet.tsx

```diff
@@ -163,40 +186,25 @@ export default function CharacterSheet({
                 <For
                   each={
                     [
-                      {
-                        label: t("strength_abbr"),
-                        score: character.data.Strengthscore,
-                        modifier: character.data.Strengthmod,
-                      },
-                      {
-                        label: t("dexterity_abbr"),
-                        score: character.data.Dexterityscore,
-                        modifier: character.data.Dexteritymod,
-                      },
-                      {
-                        label: t("constitution_abbr"),
-                        score: character.data.Constitutionscore,
-                        modifier: character.data.Constitutionmod,
-                      },
-                      {
-                        label: t("intelligence_abbr"),
-                        score: character.data.Intelligencescore,
-                        modifier: character.data.Intelligencemod,
-                      },
-                      {
-                        label: t("wisdom_abbr"),
-                        score: character.data.Wisdomscore,
-                        modifier: character.data.Wisdommod,
-                      },
-                      {
-                        label: t("charisma_abbr"),
-                        score: character.data.Charismascore,
-                        modifier: character.data.Charismamod,
-                      },
+                      "Strength",
+                      "Dexterity",
+                      "Constitution",
+                      "Intelligence",
+                      "Wisdom",
+                      "Charisma",
```

Comment:
🌱 This could probably be an Enum, but in the end, they might actually be defined in the FastAPI backend, which will cause the autogenerated client to get these types/enums for free.